CombinatorialGames/Game/PGame.lean

```diff
@@ -824,185 +823,173 @@ lemma bddBelow_of_small (s : Set PGame.{u}) [Small.{u} s] : BddBelow s := by
 
 If `x ≈ 0`, then the second player can always win `x`. -/
+@[deprecated AntisymmRel (since := "2025-01-26")]
 def Equiv (x y : PGame) : Prop :=
   x ≤ y ∧ y ≤ x
 
--- Porting note: deleted the scoped notation due to notation overloading with the setoid
--- instance and this causes the PGame.equiv docstring to not show up on hover.
-
-instance : IsEquiv _ PGame.Equiv where
-  refl _ := ⟨le_rfl, le_rfl⟩
-  trans := fun _ _ _ ⟨xy, yx⟩ ⟨yz, zy⟩ => ⟨xy.trans yz, zy.trans yx⟩
-  symm _ _ := And.symm
-
--- Porting note: moved the setoid instance from Basic.lean to here
-
 instance setoid : Setoid PGame :=
-  ⟨Equiv, refl, symm, Trans.trans⟩
+  AntisymmRel.setoid _ (· ≤ ·)
 
-theorem equiv_def {x y : PGame} : x ≈ y ↔ x ≤ y ∧ y ≤ x := Iff.rfl
+theorem equiv_def {x y : PGame} : x ≈ y ↔ AntisymmRel (· ≤ ·) x y := Iff.rfl
 
-theorem Equiv.le {x y : PGame} (h : x ≈ y) : x ≤ y :=
-  h.1
+@[deprecated AntisymmRel.le (since := "2025-01-26")]
+theorem Equiv.le {x y : PGame} (h : x ≈ y) : x ≤ y := h.le
 
-theorem Equiv.ge {x y : PGame} (h : x ≈ y) : y ≤ x :=
-  h.2
+@[deprecated AntisymmRel.ge (since := "2025-01-26")]
+theorem Equiv.ge {x y : PGame} (h : x ≈ y) : y ≤ x := h.ge
 
-theorem equiv_rfl {x : PGame} : x ≈ x :=
-  refl x
+@[deprecated AntisymmRel.rfl (since := "2025-01-26")]
+theorem equiv_rfl {x : PGame} : x ≈ x := by rfl
 
-theorem equiv_refl (x : PGame) : x ≈ x :=
-  refl x
+@[deprecated antisymmRel_refl (since := "2025-01-26")]
+theorem equiv_refl (x : PGame) : x ≈ x := by rfl
 
-@[symm]
-protected theorem Equiv.symm {x y : PGame} : (x ≈ y) → (y ≈ x) :=
-  symm
+@[deprecated AntisymmRel.symm (since := "2025-01-26")]
+protected theorem Equiv.symm {x y : PGame} : (x ≈ y) → (y ≈ x) := symm
 
-@[trans]
-protected theorem Equiv.trans {x y z : PGame} : (x ≈ y) → (y ≈ z) → (x ≈ z) :=
-  _root_.trans
+@[deprecated AntisymmRel.trans (since := "2025-01-26")]
+protected theorem Equiv.trans {x y z : PGame} : (x ≈ y) → (y ≈ z) → (x ≈ z) := _root_.trans
 
-protected theorem equiv_comm {x y : PGame} : (x ≈ y) ↔ (y ≈ x) :=
-  comm
+-- TODO: deprecate in favor of `antisymmRel_comm` once `≈` is made notation for
+-- `AntisymmRel (· ≤ ·)`.
+protected theorem equiv_comm {x y : PGame} : (x ≈ y) ↔ (y ≈ x) := comm
 
+@[deprecated antisymmRel_refl (since := "2025-01-26")]
 theorem equiv_of_eq {x y : PGame} (h : x = y) : x ≈ y := by subst h; rfl
 
 lemma Identical.equiv {x y} (h : x ≡ y) : x ≈ y := ⟨h.le, h.ge⟩
 
-@[trans]
+@[deprecated le_of_le_of_antisymmRel (since := "2025-01-26")]
 theorem le_of_le_of_equiv {x y z : PGame} (h₁ : x ≤ y) (h₂ : y ≈ z) : x ≤ z :=
-  h₁.trans h₂.1
+  h₁.trans_antisymmRel h₂
 
-instance : Trans
-    ((· ≤ ·) : PGame → PGame → Prop)
-    ((· ≈ ·) : PGame → PGame → Prop)
-    ((· ≤ ·) : PGame → PGame → Prop) where
-  trans := le_of_le_of_equiv
+instance : @Trans PGame PGame PGame (· ≤ ·) (· ≈ ·) (· ≤ ·) :=
+  inferInstanceAs (Trans (· ≤ ·) (AntisymmRel (· ≤ ·)) (· ≤ ·))
 
-@[trans]
+@[deprecated le_of_antisymmRel_of_le (since := "2025-01-26")]
```

Comment:
(note: we can drop deprecated theorems entirely now)

Reply:
I think keeping the deprecations here makes the diff a bit more readable, I'll open a new PR which gets rid of everything deprecated.